documentation/source/building.rst

```diff
@@ -301,6 +301,16 @@ Makefile-based Test Scripts
     You can usually write out such a file from the simulator's GUI.
     This is currently supported for the Mentor Questa, Mentor ModelSim and Aldec Riviera simulators.
 
+.. make:var:: TOPLEVEL_LANG
+
+    Used to inform the makefile scripts what HDL language of the toplevel design element is written in.
+    Currently it supports the values ``verilog`` for Verilog or SystemVerilog tops, and ``vhdl`` for VHDL tops.
+    This is used by simulators that support more than one interface (VPI, VHPI, or FLI) to select the appropriate interface to start cocotb.
+    It is also commonly used to deter execution of a cocotb test suite if the tests or DUT do not support being run with the selected :make:var:`TOPLEVEL_LANG`.
```

Comment:
This doesn't seem like user information - is anything other than cocotb itself going to have two copies of their code, one vhdl and one verilog?

Reply:
Potentially? Really the first few sentences are the pertinent information and this just follows. I'm not sure where else to document how to run the regression with different langs. 

Speaking of which, I just realized we don't have TOPLEVEL_LANG exempted in tox.

Reply:
That probably should go in the contribution guide, for how to run cocotb's own unit tests. I think only the first three sentences are useful here.

I suppose you could put the remaining sentences in an `.. info` box, with a comment that this is something that cocotb does in its own test-suite, not something that users actually ought to do.

Reply:
I'll add a section to the CONTRIBUTING.md about it as well as `SIM` being supported by `tox` once the `TOPLEVEL_LANG` exemption is added to tox.ini.